lampod-cli/src/main.rs

```diff
@@ -112,15 +158,19 @@ fn run(args: LampoCliArgs) -> error::Result<()> {
         radicle_term::success!("Wallet Generated, please store these words in a safe way");
         radicle_term::println(
             radicle_term::format::badge_primary("wallet-keys"),
-            format!("{}", radicle_term::format::highlight(mnemonic)),
+            format!("{}", radicle_term::format::highlight(mnemonic.clone())),
```

Comment:
```suggestion
            format!("{}", radicle_term::format::highlight(&mnemonic)),
```

What about this? we can avoid to clone it?